librz/core/canalysis.c

```diff
@@ -6393,3 +6393,34 @@ RZ_API ut64 rz_core_prevop_addr_force(RzCore *core, ut64 start_addr, int numinst
 	}
 	return start_addr;
 }
+
+RZ_API void rz_core_perform_auto_analysis(RzCore *core, RzCoreAnalysisType type) {
```

Comment:
had to move this function and related enum outside to this file in order to be able to use it directly instead of calling `rz_core_cmd("aa")` or `rz_core_cmd("aaa")` or `rz_core_cmd("aaaa")`.

Reply:
that sounds like a good idea.